browser/ui/views/tabs/brave_tab_context_menu_contents.cc

```diff
@@ -170,13 +172,20 @@ bool BraveTabContextMenuContents::IsBraveCommandIdEnabled(
       }
       return false;
     }
-    case BraveTabMenuModel::CommandShowVerticalTabs:
-      return true;
-    case BraveTabMenuModel::CommandBringAllTabsToThisWindow: {
-      return true;
-    }
     case BraveTabMenuModel::CommandCloseDuplicateTabs:
       return brave::HasDuplicateTabs(browser_);
+    case BraveTabMenuModel::CommandShowVerticalTabs:
+      [[fallthrough]];
+    case BraveTabMenuModel::CommandBringAllTabsToThisWindow:
+      [[fallthrough]];
```

Comment:
Not related to this PR, but I'm kinda surprised we don't check if there are tabs in other windows before enabling this.